stock_orderpoint_automatic_creation/models/res_company.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Daniel Campos <danielcampos@avanzosc.es> - Avanzosc S.L.
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, exceptions, fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    orderpoint_product_max_qty = fields.Integer(
```

Comment:
@cubells Why not using Product Decimal Precision here (as defined in stock.warehouse.orderpoint)?